mp/src/game/shared/neo/neo_gamerules.cpp

```diff
@@ -75,6 +75,17 @@ ConVar neo_sv_suicide_prevent_cap_punish("neo_sv_suicide_prevent_cap_punish", "1
 										 true, 0.0f, true, 1.0f);
 #endif
 
+enum ENeoCollision
+{
+	NEOCOLLISION_NONE = 0,
+	NEOCOLLISION_TEAM,
+	NEOCOLLISION_ALL,
+
+	NEOCOLLISION__TOTAL,
+};
+
+ConVar neo_sv_collision("neo_sv_collision", "0", FCVAR_REPLICATED, "0 = No collision (default), 1 = Team-based collision, 2 = All player collision", true, 0.0f, true, NEOCOLLISION__TOTAL - 1);
```

Comment:
As per discussions in discord, we may want to leave this as `FCVAR_DEVELOPMENTONLY` or `FCVAR_CHEAT` for the sake of parity.

```suggestion
ConVar neo_sv_collision("neo_sv_collision", "0", FCVAR_CHEAT | FCVAR_REPLICATED, "0 = No collision (default), 1 = Team-based collision, 2 = All player collision", true, 0.0f, true, NEOCOLLISION__TOTAL - 1);
```

Reply:
> As per discussions in discord, we may want to leave this as `FCVAR_DEVELOPMENTONLY` or `FCVAR_CHEAT` for the sake of parity.
> 
> ```suggestion
> ConVar neo_sv_collision("neo_sv_collision", "0", FCVAR_CHEAT | FCVAR_REPLICATED, "0 = No collision (default), 1 = Team-based collision, 2 = All player collision", true, 0.0f, true, NEOCOLLISION__TOTAL - 1);
> ```

Nah, it doesn't affect the current behavior by default and just putting an unneeded barrier if people want to have a test of it.